RecoParticleFlow/PFClusterProducer/src/PFClusterEMEnergyCorrector.cc

```diff
@@ -237,6 +235,21 @@ void PFClusterEMEnergyCorrector::correctEnergies(const edm::Event &evt,
     return;
   }
 
+  const double meanlimlow = -0.336;
+  const double meanlimhigh = 0.916;
+  const double meanoffset = meanlimlow + 0.5*(meanlimhigh-meanlimlow);
+  const double meanscale = 0.5*(meanlimhigh-meanlimlow);
+  
+  const double sigmalimlowEB = 0.001;
+  const double sigmalimhighEB = 0.4;
+  const double sigmaoffsetEB = sigmalimlowEB + 0.5*(sigmalimhighEB-sigmalimlowEB);
+  const double sigmascaleEB = 0.5*(sigmalimhighEB-sigmalimlowEB);  
+
+  const double sigmalimlowEE = 0.001;
+  const double sigmalimhighEE = 0.1;
```

Comment:
I don't fully understand the suggestion: you want me to just change the EE numbers for the new corrections? I think that would make the code more difficult to read. I think it is a small copy-paste, but improves a lot the readability.

If you insist, I am happy to break the definitions, but I do think it will make more cumbersome to read.

Please, let me know.